.github/workflows/build.yml

```diff
@@ -20,13 +20,13 @@ jobs:
       
       - name: Run local test Server
         run: |
-          docker pull victorrds/etebase
-          docker run -d -v $PWD/scripts:/etebase/scripts -e DEBUG=true -p 80:3735 victorrds/etebase ./scripts/testmode.sh
-          ./scripts/wait-for-it.sh localhost:80
+          #docker run -p 80:3735 etesync/test-server:0.7.0
+          docker run -d -p 3735:3735 djmitche/etesync-test-server:0.7.0
+          ./scripts/wait-for-it.sh localhost:3735
           
       - name: Tests
         env:
-          ETEBASE_TEST_HOST: localhost:80
+          ETEBASE_TEST_SERVER: localhost:3735
```

Comment:
Why rename the env var?

Reply:
I spent a bit trying to figure out how to customize the port, before realizing that this was not just a hostname.  I can change it back..